protected/humhub/modules/file/models/File.php

```diff
@@ -112,12 +139,37 @@ public function behaviors()
     }
 
     /**
-     * Gets query for [[FileHistory]].
+     * @return array
+     * @noinspection PhpMissingReturnTypeInspection
+     * @noinspection ReturnTypeCanBeDeclaredInspection
+     */
+    public function transactions()
```

Comment:
We are close to the last beta release and stable release. I don't want to enable transactions in this stage.

Reply:
disabled in https://github.com/humhub/humhub/pull/6590/commits/3480969ae6a0410ae5aa5ac78f3eed6a0d64b98b. See https://github.com/humhub/humhub/pull/6590#discussion_r1335063469 for more info.